scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/ModelData.scala

```diff
@@ -58,16 +58,24 @@ object ModelData extends LazyLogging {
   // Also a classloader is correct so we don't need to build the new one
   // This tiny method is Flink specific so probably the interpreter module is not the best one
   // but it is very convenient to keep in near normal, duringExecution method
-  def duringFlinkExecution(inputConfig: Config): ModelData = {
-    duringExecution(inputConfig, ModelClassLoader.empty, resolveConfigs = false)
+  def duringFlinkExecution(
+      inputConfig: Config,
```

Comment:
maybe we could introduce some new class (with well-named fields) to gather all the configs?